gitops/components/kubevirt/kustomize/kustomization.yaml

```diff
@@ -28,4 +33,12 @@ patches:
                 operator: Exists
               - key: pelo.tech/uki-labs
                 operator: Exists
-
+  - target:
+      group: apps
+      version: v1
+      kind: Deployment
+      name: virt-operator
+    patch: |-
+      - op: replace
```

Comment:
From another comment this is a json6902 limitation which I think is watch patches uses. so for future proofing we'd want to add something like
```
      - op: test
        path: /spec/template/spec/containers/0/env/0/name
        value: VIRT_OPERATOR_IMAGE
        ```
   Which will fail if the value changes